fhir-server-test/src/test/java/com/ibm/fhir/server/test/profiles/DavinciHealthRecordExchangeTest.java

```diff
@@ -440,6 +443,7 @@ public Parameters getInputParameters() {
                                     .build());
                     Parameters.Parameter coverageParameter = coverageParameterBuilder.build();
                     params.add(coverageParameter);
+                    params.add(source.getParameter().get(3));
```

Comment:
I expected line 75 to be updated with a new set of profiles. Were there no additional profiles? or removals of profiles?

Reply:
There was no removal of profiles -- there was one added:  "http://hl7.org/fhir/us/davinci-hrex/StructureDefinition/hrex-claimresponse|0.2.0"

I added that to line 75.